src/slurm_plugin/clustermgtd.py

```diff
@@ -911,6 +932,7 @@ def _handle_bootstrap_failure_nodes(self, active_nodes):
             log.warning("Found the following bootstrap failure nodes: %s", print_with_count(bootstrap_failure_nodes))
             # Increase the partition failure count and add failed nodes to the set of bootstrap failure nodes.
             self._increase_partitions_protected_failure_count(bootstrap_failure_nodes)
+        self._event_publisher.publish_bootstrap_failure_events(bootstrap_failure_nodes)
```

Comment:
What about moving the function inside ` if bootstrap_failure_nodes:`?

Reply:
As we spoke about, I would like to provide at least some mechanism to differentiate no errors from no signal. With the way it is now, a user can configure the log level to debug and make this differentiation. This might not be a valid design though, so let me know and I can change it.